python/src/aica_api/client.py

```diff
@@ -54,7 +59,56 @@ def _endpoint(self, endpoint=''):
         """
         if self._protocol is None:
             self.protocol()
-        return f'{self._address}/{self._protocol}/{endpoint}'
+        return self.__raw_endpoint(f'{self._protocol}/{endpoint}')
+
+    def __raw_endpoint(self, endpoint: str) -> str:
+        return f'{self._address}/{endpoint}'
+
+    def __create_token(self) -> None:
```

Comment:
nit: maybe `__ensure_token` since it only requests and creates one if it is not None?